scripts/workers/worker_util.py

```diff
@@ -54,8 +54,8 @@ def delete_message_from_sqs_queue(self, receipt_handle):
         url = URLS.get("delete_message_from_sqs_queue").format(self.QUEUE_NAME)
         url = self.return_url_per_environment(url)
         data = {"receipt_handle": receipt_handle}
-        response = self.make_request(url, "POST", data)  # noqa
-        return response.status_code
+        response = self.make_request(url, "POST", data)
+        return response
```

Comment:
Please remove this change, as it is being handled in https://github.com/Cloud-CV/EvalAI/pull/2575

Reply:
@vkartik97 :+1: I will also remove the test for `delete_message_for_sqs_queue`. It will cause an error without the correction so I think we can add it after the fix in #2575 is applied.

Reply:
Sure 💯

Reply:
@vkartik97 Done! 👍 Also verified that the Travis CI is passing.